source/extensions/filters/http/header_to_metadata/header_to_metadata_filter.cc

```diff
@@ -14,7 +15,7 @@ namespace HttpFilters {
 namespace HeaderToMetadataFilter {
 namespace {
 
-const uint32_t MAX_HEADER_VALUE_LEN = 100;
+const uint32_t MAX_HEADER_VALUE_LEN = 10 * 1024;
```

Comment:
This is a bit large, isn't it? I think 8k is a de-factor bound.

Reply:
Updated to 8k.